l10n_it_sdi_channel/views/sdi_view.xml

```diff
@@ -45,11 +46,11 @@
             </field>
         </record>
         <record id="action_sdi_channel" model="ir.actions.act_window">
-            <field name="name">SdI Channel</field>
+            <field name="name">ES Channel</field>
```

Comment:
Come hai cambiato la label qui e anche un po' più sotto?

Reply:
facevano parte del commit che ho cherry-picckato, le ho tolte visto che non hanno senso così

Reply:
Avevano senso invece.
Guarda la pagina in inglese di fatturapa.gov.it:

https://www.fatturapa.gov.it/export/fatturazione/en/sdi.htm

Reply:
Ah ecco! Grazie @primes2h.